portfolio-project-react/src/components/About/About.tsx

```diff
@@ -1,27 +1,41 @@
-import { useContext, useEffect, useState } from "react";
-import { IsEditModeEnabled } from "../../EditModeContext";
+import React, { useContext, useEffect, useState, FC } from "react";
+import { IsEditModeEnabled } from "../../EditModeContext.tsx";
 import {
   ABOUT_HEADING,
   CONTACT_HEADING,
   EMAIL_HEADING,
   CONTACT_PLACEHOLDER,
   EMAIL_PLACEHOLDER,
-} from "./constants";
+} from "./constants.tsx";
 import styles from "./About.module.css";
 
-const About = ({ totalWorkExperience }) => {
+
+
+interface AboutComponentProps {
+  totalWorkExperience: {
```

Comment:
Not sure but you won't be needing this interface nesting. You can directly initiate the types under AboutComponentProps. Unless you have done this for some specific functionality.